//package com.sincetimes.redwar.game.comnunicate.protocol;

message RseUpdateProfile
{
	optional int32 ret = 1;

}